src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+
+        if (userService.findByEmail(email) != null) {
+            User user = userService.findByEmail(email);
+            if (user.getPassword().equals(password)) {
+                return true;
+            }
+            return false;
+        }
         return false;
```

Comment:
userService.findByEmail(email) - you can save it to variable and use it many times